<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
    <summary errors="0" failures="1" finished="true" name="Test Run: 16" tests="2" total_time="Finished in: 0.00 secs."/>
    <test location="/Users/jeff/Documents/workspaces/dme/track_logger_converter/tests/parsers/can_frame_parser_test.py" status="ok" test="CanFrameParserTest.testFrame242_1" time="0.00">
        <stdout>
            <![CDATA[Clutch: 0
RPM: 0
ECU_THROTTLE: 0
]]>
        </stdout>
        <stderr/>
    </test>
    <test location="/Users/jeff/Documents/workspaces/dme/track_logger_converter/tests/parsers/can_frame_parser_test.py" status="fail" test="CanFrameParserTest.testFrame242_2" time="">
        <stdout>
            <![CDATA[Clutch: 8
RPM: 1355
ECU_THROTTLE: 76
]]>
        </stdout>
        <stderr>
            <![CDATA[Traceback (most recent call last):
  File "/Users/jeff/Documents/workspaces/dme/track_logger_converter/tests/parsers/can_frame_parser_test.py", line 30, in testFrame242_2
    self.assertEqual(5420, self.test_state.get_data_item(DataState.get_data_name_at_idx(DataState.names.RPM)), 'Engine RPM was not parsed correctly.')
AssertionError: Engine RPM was not parsed correctly.
]]>
        </stderr>
    </test>
    <launch mode="run">
        <launch_memento>
            <![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="track_logger_converter can_frame_parser_test.py"/>
]]>
        </launch_memento>
    </launch>
</pydev-testsuite>
